### DISCUSSION TITLE
请问这个可以在微信小程序中使用吗？

### DISCUSSION THREAD

**Discussion by fighxp** [plainheart/echarts-extension-amap#21] [Q&A] [answered]
作者你好，
应用背景：在小程序中使用echarts基于高德地图渲染图表，
操作过程：在小程序中
1.npm inti 
2.npm install echarts-extension-amap --save 安装了这个包，
3.开启小程序本地使用npm
4.小程序工具——构建
5.需要用到此包的js文件中
// 使用 require
const echarts = require('echarts');
const echartsExtensionAmap = require('echarts-extension-amap');
但是构建完后一直提示文件no defined，请问下这个包支持微信小程序吗？感谢您的回答。

我按照这个安装顺序，安装了微信小程序上其他包，操作步骤和引用方式和上方一样未出现报错。

**Comment by plainheart** [comment, marked as answer]
抱歉，这个不支持，高德地图依赖 DOM，而小程序不支持操作 DOM。

**Reply by fighxp** [reply]
感谢您的解答
